discovery-provider/src/tasks/update_aggregates.py

```diff
@@ -39,21 +37,21 @@
     select
         ap.playlist_id,
         coalesce(ps.save_count, 0) as save_count,
-				coalesce(pr.repost_count, 0) as repost_count
+        coalesce(pr.repost_count, 0) as repost_count
     from
```

Comment:
maybe want to reformat this file to match main before merging?

Reply:
oh the format on main had lint errors so i fixed. @isaacsolo that ok?

Reply:
yup ty

Reply:
ah got it, so this is the correct formatting